.storybook/styles/application.scss

```diff
@@ -48,10 +48,8 @@ $output-bourbon-deprecation-warnings: false;
 @import 'components/progress-circle';
 @import 'components/login';
 @import 'components/pagination';
-@import 'components/sidebar-content';
```

Comment:
I noticed a couple style files were no longer being used. Not sure if they should still be there for reference?

Reply:
I'm fine with deleting them!